### DISCUSSION TITLE
How to use it in a Vue js + Capacitor application?

### DISCUSSION THREAD

**Discussion by volkanelci1** [capacitor-community/camera-preview#251] [Q&A]
I want to use this plugin in my Vue js application.
I tried to follow the documentation but it did not work.
Can someone please help me with this?
Thanks

**Comment by pbowyer** [comment]
Hi @volkanelci1,

What problems are you having? I have an Ionic Vue application running on iOS and Android and it works great 😁

**Reply by volkanelci1** [reply]
My application is not an Ionic application. It started as a web project but then I decided to make it an app with Capacitor.js
When I follow the steps in the documentation I get errors because of typescript. I don't use typescript in my application

**Reply by pbowyer** [reply]
Can you try the changes in pull request #249 which may sort that?

If they don't , please post the errors in a new issue. Thanks!

**Reply by volkanelci1** [reply]
Here is my Vue component

```
<template>
  <div>
    
  </div>
</template>

<script>
import '@capacitor-community/camera-preview'
import { CameraPreview } from '@capacitor-community/camera-preview';

export default {
  data() {
    return {
      cameraPreviewOptions: {
        position: 'rear',
        height: 1920,
        width: 1080
      }
    }
  },
  created() {
    CameraPreview.start(this.cameraPreviewOptions);
  }
}
</script>
```

I get this error on console. Can you please tell me what am I doing wrong here
https://ibb.co/nrmrJFq

**Reply by VanIseghemThomas** [reply]
Same question and error. Is it mandatory to use Ionic? Just got into Capacitor so still learning.
